requirements.txt

```diff
@@ -1,32 +1,31 @@
-pyquil>=2.14
-numpy
-networkx
-pandas
-lmfit
-scipy
-sympy
-python-rapidjson
-seaborn
-cvxpy>=1.0.0
-dataclasses; python_version < "3.7"
-tqdm
-gitpython
-matplotlib>=3.1.0
+pyquil>=3.0.0,<4.0.0
+numpy==1.20.*
```

Comment:
Locked down these as CI was taking a horrendous amount of time to run (it had to download all available versions of each dependency)